lms/templates/components/tabbed/tabbed_view.underscore

```diff
@@ -1,5 +1,5 @@
 <div class="page-content">
     <nav class="page-content-nav" role="tablist"></nav>
     <div class="sr-is-focusable" tabindex="-1"></div>
-    <div class="page-content-main"></div>
+    <div class="page-content-main" aria-live="polite"></div>
```

Comment:
Making this an aria-live region will force a screen reader to read all the text injected into this, which I don't think we want to do.  Moving focus here is good enough, the screen reader user can then consume the info at their own pace.